lib/pbench/client/oidc_admin.py

```diff
@@ -136,12 +138,12 @@ def get_user(self, username: str, token: str) -> dict:
             }
         """
         response = self.get(
-            f"admin/realms/{self.OIDC_REALM}/users?username={username}",
+            f"admin/realms/{self.OIDC_REALM}/users",
             headers={
                 "Content-Type": "application/json",
                 "Authorization": f"Bearer {token}",
             },
-            verify=False,
+            username=username,
```

Comment:
This made me look through the `get` and `_method`. Yeah, it seems that any "loose" keywords here get bundled into request `params`, but I'm kinda surprised that nobody complained about how `Connection._method` accepts `**kwargs` and then uses that as one of several parameters to overwrite the local definition of `kwargs` as a `dict` of keyword parameters to `request`. Wow; that's twisted. Using a different name would have been better: it's really easy when skimming through to assume that the `**kwargs` in the request call is the same `kwargs` passed into `_method`. Ah well; that's not really in scope for this PR.

Reply:
Yes, I did realize that. I think it should not add the value of a `param` passed as `kwargs`. The `kwargs` should be a separate argument to the `dict`.

Reply:
Opened an issue https://github.com/distributed-system-analysis/pbench/issues/3253

Reply:
> I'm kinda surprised that nobody complained

Ya mean like [this](I'm kinda surprised that nobody complained)?

I'm [sure](https://github.com/distributed-system-analysis/pbench/pull/3114#issuecomment-1421507100) that @portante will fix it.

Reply:
Somehow I'd completely forgotten that one. Good bookkeeping. Yeah, now it'll never be changed. 😆

Reply:
No, no, we have an _issue_ for it now.  I'm sure it will be addressed.  🙂